apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/types.ts

```diff
@@ -0,0 +1,8 @@
+export type TourAsset = {
+	desktop: { src: string; type: string };
+	mobile?: { src: string; type: string };
+};
+
+export type TourAssets = {
```

Comment:
Since we don't use this anywhere, I'd avoid introducing it just yet. Also, we may decide to rethink the index signature once tour contents are migrated to TS - might be the case a closed-form or mapping would be more appropriate.